keel-web/src/main/kotlin/com/netflix/spinnaker/keel/services/ApplicationService.kt

```diff
@@ -38,14 +38,11 @@ import org.springframework.stereotype.Component
 @Component
 class ApplicationService(
   private val repository: KeelRepository,
-  private val constraintEvaluators: List<ConstraintEvaluator<*>>
+  constraintEvaluators: List<ConstraintEvaluator<*>>
```

Comment:
just out of curiosity, why was `private val` removed?

Reply:
IntelliJ told me this wasn't accessed as a property anywhere (and, in fact, it's only used to build `statelessContraintEvaluators` below).